pkg/errors/comparison.go

```diff
@@ -29,7 +31,7 @@ func Resemble(a, b error) bool {
 	if !ok {
 		return false
 	}
-	return ttnA.Code() == ttnB.Code() &&
+	return (ttnA.Code() == uint32(codes.Unknown) || ttnB.Code() == uint32(codes.Unknown) || ttnA.Code() == ttnB.Code()) &&
```

Comment:
I don't really understand this. It shouldn't be possible to define an error with the same name in the same namespace, but give it a different code.

Reply:
The big question is: should https://github.com/TheThingsNetwork/lorawan-stack/pull/2941/files#diff-4ba598240540ba3699477b82237d81b5R59 `BeTrue` or `BeFalse`. I think it makes more sense for this to be true.

Reply:
Okay. Then maybe we should just only compare the namespace and name.

Reply:
Also fine with me